utils/helpers.js

```diff
@@ -30,9 +30,121 @@ function stripNoMinifyBlocks(nodes) {
   }).reduce((a, b) => a.concat(b), []);
 }
 
+function getElementAttribute(node, attrName) {
+  let attribute = node.attributes.find((attr) => {
+    return attr.type === 'AttrNode' && attr.name === attrName;
+  });
+  return (attribute || {}).value;
+}
+
+function isClassIncluded(chars, classes) {
+  chars = (chars || '').trim().split(' ');
+
+  return chars.some((char) => {
+    return classes.indexOf(char) !== -1;
+  });
+}
+
+function canTrimUnnecessaryWhiteSpace(value, config) {
```

Comment:
the method name should in some way suggest that this is handling values of the `class` property

Reply:
do we need to pass in the complete `config` here or would `config.classes` be enough?